db/db_common_test.go

```diff
@@ -13,30 +12,20 @@ import (
 )
 
 func testBusyTimeout(t *testing.T, db *DB) {
-	rbt := random.Intn(10000)
-	_, err := db.ExecuteStringStmt(fmt.Sprintf("PRAGMA busy_timeout=%d", rbt))
+	wantTimeout := random.Intn(10000)
```

Comment:
The best way to test this is to generate two different random numbers, one for each timeout, and then read them back. Make sure they line up. Agreed?

Reply:
Yes, I just pushed that. Thanks!